packages/remix-server-runtime/crypto.ts

```diff
@@ -5,59 +5,57 @@ export type UnsignFunction = (
   secret: string
 ) => Promise<string | false>;
 
-// TODO: Once node v16 is available on AWS we should use the globally provided
-// webcrypto "crypto" variable and re-enable this code-path in "./cookies.ts"
-// instead of referencing the sign and unsign globals.
+// TODO: Once Node v19 is supported we should use the globally provided
+// Web Crypto API's and re-enable this code-path in "./cookies.ts"
+// instead of referencing the `sign` and `unsign` globals.
 
 // const encoder = new TextEncoder();
 
-// export async function sign(value: string, secret: string): Promise<string> {
-//   let key = await crypto.subtle.importKey(
-//     "raw",
-//     encoder.encode(secret),
-//     { name: "HMAC", hash: "SHA-256" },
-//     false,
-//     ["sign"]
-//   );
-
```

Comment:
Why move from `async function sign` to an arrow function?  I personally think the industry got too enamored with arrow functions.  If it's not an tiny/inline function, a predicate, or using `this` - I strongly prefer `function`.

Reply:
I just copied over what we have in Cloudflare & Deno 🤷‍♂️